src/config/normalize/index.ts

```diff
@@ -179,6 +184,9 @@ async function normalizedSimpleConfig(
         ...(commonOptions.mode || config.compilation?.mode
             ? { mode: commonOptions.mode || config.compilation?.mode }
             : {}),
+        ...(commonOptions.sourcemap ?? config.compilation?.sourcemap
+            ? { sourcemap: organizeSourcemap(commonOptions.sourcemap ?? config.compilation?.sourcemap) }
```

Comment:
move `organizeSourcemap` [here](https://github.com/farm-fe/farmup/pull/23/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R126) would be a better choice